Doc/library/types.rst

```diff
@@ -98,6 +98,10 @@ the types that arise only incidentally during processing such as the
 Typical use of these names is for :func:`isinstance` or
 :func:`issubclass` checks.
 
+.. note::
```

Comment:
Delete this and just add a revised sentence (next comment) to the previous paragraph.
```suggestion

```